_config.yml

```
@@ -394,7 +394,7 @@ vendors:
     pace: npm/pace-js@1.0.2/pace.min.js # ok
     pjax: npm/pjax@0.2.8/pjax.min.js # ok
     fetch: npm/whatwg-fetch@3.4.0/dist/fetch.umd.min.js # ok
-    anime: npm/animejs@3.2.0/lib/anime.min.js # ok
+    anime: npm:theme-shokax-anime@latest/anime.shokax.min.js # ok
```

```suggestion
    anime: npm/theme-shokax-anime@latest/anime.shokax.min.js
```
目前的vendors需要用shoka的原有格式，因为还没给vendors加srcHelper函数